docs/modules/layers.rst

```diff
@@ -163,6 +163,8 @@
     LocalResponseNormalization2DLayer
     BatchNormLayer
     batch_norm
+    InstanceNormLayer
+    instance_norm
```

Comment:
This doesn't match the code any more. Should have StandardizationLayer, instance_norm and layer_norm instead.